### PR TITLE
add platform is cordova check for ionic serve / browser usage

### PR DESCRIPTION
**Pull request by dmastag**

### CODE REVIEW

File: src/providers/image-loader.ts [hunk 1]
```diff
@@ -78,16 +78,25 @@ export class ImageLoader {
...
-    Observable.fromEvent(document, 'deviceready').first().subscribe(res => {
-      if (this.nativeAvailable) {
-        this.initCache();
-      } else {
+    platform.ready().then(() => {
+      if (!platform.is('cordova')) {
         // we are running on a browser, or using livereload
         // plugin will not function in this case
-        this.isInit = true;
+        this.init = true;
```
Comment (ihadeed) [inline src/providers/image-loader.ts]: Why did you change this `isInit` to `init` but the other instances still read `isInit`

Comment (dmastag) [inline src/providers/image-loader.ts]: Ahh, i just copied pasted from your comment here
https://github.com/zyra/ionic-image-loader/issues/77#issuecomment-315415843

**Comment by dmastag** [comment]
Fixed

**Comment by ihadeed** [comment]
thanks!
